app/actionReorder/post.controller.js

```diff
@@ -0,0 +1,53 @@
+const { body } = require('express-validator')
+const { logger } = require('../../common/logging/logger')
+const { setActionPriorities } = require('../../common/data/sentencePlanningApi')
+const { catchAndReThrowError } = require('../../common/utils/util')
+
+const getActionRowKeys = bodyData => Object.keys(bodyData).filter(element => element.match(/^[0-9a-f-]{1,}$/g))
+
+const validationRules = () => {
+  return [
+    body()
+      // Check that all the priority index values are integers
+      .custom(value =>
+        getActionRowKeys(value).reduce((soFar, element) => soFar && Number.isInteger(Number(value[element])), true)
+      )
+      .withMessage('Priority values should all be integers')
+      // Check all the priority index values are unique
+      .custom(value =>
+        getActionRowKeys(value)
+          .map(element => Number(value[element]))
+          .reduce((soFar, element, index, src) => soFar && src.lastIndexOf(element) === index, true)
+      )
+      .withMessage('Priority values are not unique'),
```

Comment:
Ditto comment above - these aren't really user-facing error messages. Should we actually throw a proper error in these cases?